sbndaq-artdaq-core/Overlays/ICARUS/ICARUSTriggerUDPFragment.hh

```diff
@@ -2,10 +2,13 @@
 #define sbndaq_artdaq_core_Overlays_ICARUS_ICARUSTriggerUDPFragment_hh
 
 #include "artdaq-core/Data/Fragment.hh"
+#include "sbndaq-artdaq-core/Trace/trace_defines.h"
```

Comment:
Do we need this in the offline branch? If not, I'd suggest removing the trace include here to avoid picking up some dependency.